xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/editactions.vm

```diff
@@ -37,6 +37,12 @@
     ## This allows the preview form to redirect back to the correct editor when pressing "Back to edit":
     <span class="meta-versionSummary metamodifs">
       <input type="hidden" name="xeditaction" value="$!xcontext.action" />
+      ## Current version of the document for the conflict check
+      <input type="hidden" id="previousVersion" name="previousVersion" value="$doc.version" />
+        ## If the document is new or not when the user started the edition, for the conflict check too.
```

Comment:
This line is indented in order to be seen as a "child" of the previous comment? It's a bit strange that the next line with the input is not indented.

Reply:
I think it's an indent created when I copy/pasted the code there. Will fix it.